master/buildbot/monkeypatches/__init__.py

```diff
@@ -58,6 +58,26 @@ def patch_testcase_assert_raises_regexp():
 
 
 @onlyOnce
+def patch_mysqlclient_warnings():
+    try:
+        from _mysql_exceptions import Warning
+        # MySQLdb.compat is only present in mysqlclient
+        import MySQLdb.compat  # noqa pylint: disable=unused-import
+    except ImportError:
+        return
+    # workaround for https://twistedmatrix.com/trac/ticket/9005
+    # mysqlclient is easier to patch than twisted
+    # we swap _mysql_exceptions.Warning arguments so that the code is in second place
+
+    def patched_init(self, *args):
+        if isinstance(args[0], long):
```

Comment:
@tardyp I just noticed this change that you did.  This causes a problem on Python 3, because
there is no `long` type there.  Can you fix this?

As you are introducing new patches, can you run the tests in a Python3 virtualenv, just to double check
that new Python 3 regressions are being introduced as we move forward?

Thanks.